src/cacheSeed/cacheSeedManager.ts

```diff
@@ -168,6 +167,8 @@ export class CacheSeedManager {
         percentage: 100,
       });
     } catch (error) {
+      const errorObj = { jobId, taskId, msg: 'Reject task and increase attempts', error };
```

Comment:
i think all errors should be "error" or "err" can't be both.
in error log you can pass the error object separatably as is done in all services

Reply:
done